docs/lib/auth/fragments/android/signin/60_check_sign_in_state.md

```diff
@@ -0,0 +1,55 @@
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.Auth.fetchAuthSession(
```

Comment:
Let's wrap this in a method that returns a boolean, since a customer would likely want to do the same anyway.  Something like:
```
private boolean isAuthenticated() {
    ...
}
```

Reply:
Is that necessary? The code snippets provided in getting started guide are for customers to call in `onCreated()` method

Reply:
Now that I think about it, we can't actually return a boolean since `fetchAuthSession` is async, so you can disregard this!

Reply:
Now that I think about it, we can't actually return a boolean since `fetchAuthSession` is async, so you can disregard this!